gobblin-runtime/src/main/java/gobblin/runtime/mapreduce/MRJobLauncher.java

```diff
@@ -182,6 +182,7 @@ public void close()
         cleanUpWorkingDirectory();
       } finally {
         super.close();
+        fs.close();
```

Comment:
I think it is good practice for non-HDFS file systems. However, this is the fs that supports MR, which must be either HDFS or local file system afaik, so I don't see a benefit to using a new instance of the filesystem.